com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -333,7 +339,7 @@ public static void NetworkHide(List<NetworkObject> networkObjects, ulong clientI
 
         private void OnDestroy()
         {
-            if (NetworkManager.Singleton != null && NetworkManager.Singleton.SpawnManager.SpawnedObjects.ContainsKey(NetworkObjectId))
+            if (NetworkManager.Singleton != null && NetworkManager.Singleton.SpawnManager != null && NetworkManager.Singleton.SpawnManager.SpawnedObjects.ContainsKey(NetworkObjectId))
```

Comment:
I see you found the same bug I found earlier where the SpawnManager no longer exists but objects are still being destroyed.